ROADMAP.md

```diff
@@ -1,103 +1,104 @@
 
-### Implement the functionality basically
+* Support for more deployment options
+* Support multi-server computing framework.
+* Development of operation and maintenance tools.
 
-In this release, we've implemented most of the functionality and some advanced features of `Dragonfly` with `Go`, to the point of being aligned with the Java version. Including:
+We will try to finish it before **December 30, 2019**.
 
-* Scheduler. Complete scheduling logic, including blacklist or something others. In addition, for better extensibility, we should support it as a plug-in.
-* CDN support. Use CDN to Cache downloaded data from source to avoid downloading same files repeatedly.
-* Dynamic downloading rate limiting.
-* Disk GC.
-* Preheat image.
+### Support for more deployment options
 
-## v0.5.0
+As a cloud native project, we should do more work to support deploy the `Dragonfly` on the kubernets platform. Including but not limited to the following list:
 
-This is a basically stable version, reaching the pre-GA stage. Including:
+* Deploy `supernode` using [Helm](https://github.com/helm/helm) in Kubernetes to simplify the complexity of scaling SuperNodes in Kubernetes.
+* Deploy `supernode` cluster using [Operator](https://coreos.com/operators/).
+* Deploy `dfget & dfdaemon` using DaemonSet in Kubernetes.
 
-1. Stability & Security enhancement.
-2. Function Enhancement.
+Related issue: [#346](https://github.com/dragonflyoss/Dragonfly/issues/346)
 
-We will try to finish it before **April 15, 2019**.
+### Support multi-server computing framework
 
-### Stability & Security enhancement
+With the rise of ARM servers, x86 servers will not be the only choice.
+However, the images of the two computing frameworks are incompatible for one application.
+So the user will have to maintain the different versions for the same image which will bring challenges to management.
 
-Enhance the stability of the project.
+Related issue: [#775](https://github.com/dragonflyoss/Dragonfly/issues/775)
 
-* Support more configuration items.
-* Support private container image.
-* Support authentication in SuperNode API.
-* Different encryption algorithm in data transmission.
+### Development of operation and maintenance tools
 
-### Function Enhancement
+For better to use `Dragonfly`, we plan to provide more operation and maintenance tools to make it easy, including:
 
-Add more advanced functionality on the basis of previous versions.
+* A tool to validate the config file.
+* A tool for performance testing of `Dragonfly` clusters.
+* A dashboard tool for `Dragonfly`.
 
-* Support multi-registrys.
-* Support supernode cluster. Cluster the SuperNode to decrease possibility of failure.
-* Supernode support storage driver.
-* Use [IPFS](https://github.com/ipfs/go-ipfs) to share block datas between SuperNodes.
-* Highly user-customized modules.
+## v1.0.x
 
-## Future
+This is a stable version in GA stage. And we will focus on making `Dragonfly` support more scenarios. Including:
```

Comment:
I assume when we say GA, it means there is no user impact going forward: no behavior change, features are not going to be deprecated, etc. 

I feel somehow, we should track all the features stage somehow, not all of them can go GA at the same time. For example, the core it pretty stable, but the plugin framework might still be in alpha.